docs/frameworks.md

```diff
@@ -111,6 +111,34 @@ exports.config = {
 };
 ```
 
+Using Serenity/JS
+-----------------
+
+[Serenity/JS](http://serenity-js.org) is an acceptance testing library which can be integrated as a drop-in replacement of [Mocha](http://serenity-js.org/mocha/readme.html) or [Cucumber](http://serenity-js.org/cucumber/readme.html) framework adapters to provide advanced [scalability and reporting capabilities](http://serenity-js.org/overview/readme.html).
+
+To use it, [install](http://serenity-js.org/overview/installation.html) and [configure](http://serenity-js.org/overview/configuration.html) your test framework of choice.
+
+Next, [install Serenity/JS](http://serenity-js.org/overview/installation.html):
```

Comment:
Install is mentioned the installation link is mentioned here and in the previous line do you want both of these instances?